index.js

```diff
@@ -1,33 +1,39 @@
 
-
-//Getting the different placeholders
-
 let content = document.getElementById("content");
 let integerValue = parseInt(content.innerHTML);
 
-const decreaseButton = document.getElementById("btn-decrease");
-const resetButton = document.getElementById("btn-reset");
-const increaseButton = document.getElementById("btn-increase");
+const arrayButton = document.querySelectorAll(".button-event");
+arrayButton.forEach((button, index) => {
+    button.addEventListener("click", () => {
+        changeValue(index)
+        })
+    }
+)
+
+function changeValue(index) {
+    switch (index) {
+        case 0:
+            integerValue -= 1;
+            break;
+
+        case 1:
+            integerValue = 0;
+            break;
+
+        case 2:
+            integerValue += 1;
+            
```

Comment:
You missed a break here and there should always have a default case
Otherwise than this you did a great job 🔥